### PR TITLE
Increased lighthouse performance

### LINKED ISSUE #1575: Improve lighthouse performance of generated webapp.
Lighthouse performance should be increased.

lighthouse

**Would you like to work on the issue?**

Yes.

### PR DESCRIPTION
**Pull request by agbilotia1998**
Fixes #1575 #1211

Preview Link:https://agbilotia1998.github.io/Fossasia14/

Screenshots for the change: 
improved

### CODE REVIEW

**Comment by ParthS007** [comment]
@agbilotia1998 I was working on that :worried:

**Comment by agbilotia1998** [comment]
Oh okay, but still there are accessibility related performance issues you can work on them.

File: src/backend/dist.js [hunk 1]
```diff
@@ -365,7 +365,7 @@ module.exports = {
 
     try {
       var fileData = fs.readFileSync(filePath).toString().split('\n');
-      fileData.unshift("{\n\"short_name\":\"" + eventName+"\",");
+      fileData.unshift("{\n\"name\":\""+eventName+"\",\n\"short_name\":\"" + eventName+"\",");
```
Comment (aayusharora) [inline src/backend/dist.js]: > fileData.unshift("{\n\"name\":\""+eventName+"\",\n\"short_name\":\"" + eventName+"\",");
Why do we need to write it as `name:` @agbilotia1998

Comment (agbilotia1998) [reply src/backend/dist.js]: Please check the issue #1575 and the attached screenshot.

**Comment by agbilotia1998** [comment]
@geekyd please review.

**Comment by agbilotia1998** [comment]
@geekyd please run travis build again here.

File: src/backend/dist.js (hunk 1 above)
Comment (geekyd) [reply src/backend/dist.js]: @aayusharora We are dynamically adding the name of the event to manifest.Please check screenshot
screen shot 2018-01-09 at 12 37 24 am

**Comment by agbilotia1998** [comment]
@geekyd are any other changes required in this PR.

**Comment by geekyd** [comment]
@agbilotia1998 It looks good to me @aayusharora Please have a look.

**Comment by agbilotia1998** [comment]
@geekyd please run the travis build again.

**Comment by mariobehling** [comment]
Ok, thank you. Please continue observing this and discuss with mentors if there are additional improvements possible. Thank you! Good job!

**Pull request merged**